package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "main": "server/server.js",
   "scripts": {
-    "pretest": "jshint ."
+    "pretest": "jshint .",
+    "start": "node server/server",
```

Comment:
Nitpick: `node .` would be better, there is no need to duplicate `server/server` - see L4 `"main": "server/server.js"`